dynamic_instrumentation/src/instrument.rs

```diff
@@ -256,6 +256,13 @@ impl<'tcx> Visitor<'tcx> for InstrumentationAdder<'_, 'tcx> {
                     .debug_mir(location)
                     .add_to(self);
             }
+            Rvalue::Use(Operand::Constant(..))=> {
```

Comment:
```suggestion
            Rvalue::Use(Operand::Constant(..))=> {
                // Track (as copies) assignments that give local names to constants so that code
                // taking references to said constants can refer to these assignments as sources
```

Reply:
I think this is necessary to explain why this change is correct--otherwise, it isn't very clear why we would instrument such statically-known behavior. Is this explanation sensical?

Reply:
Based on the other comment on the diff, I think this is for future `AddrOfStatic` support.  In the MIR, taking the address of a static looks like `_1 = const {alloc1: &usize};`, which is `Use(Constant(..))` with a constant of type `&'static usize` whose value is a pointer to a static allocation.

Reply:
OK, then let's have a comment to that effect.